src/app/dynamic-forms/models/cinchy-form.model.ts

```diff
@@ -417,55 +529,51 @@ export class Form {
 
               break;
             case "Yes/No":
-              params[paramName] = field.value;
+              params[paramName] = field.value || false;
```

Comment:
This might be a product question for Michael/Borna, today Cinchy's yes/no columns support null values, do we want to preserve the nulls?

Not sure if we have users that use the null value, but I can see in some scenarios where they'd want to keep or use null in the field